experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/public/config/variables.json

```diff
@@ -31044,9 +31044,9 @@
         },
         "sdg/VC_DTH_TONCVN": {
             "dcid": "sdg/VC_DTH_TONCVN",
-            "name": "Number of conflict-related deaths",
+            "name": "Number of conflict-related deaths (non-civilians)",
```

Comment:
@dwnoble  Do these changed need to be made in the UN code repo?

Reply:
Yes - i can add it to the UN site